design-assets/README.md

```diff
@@ -0,0 +1,17 @@
+# CMSDS Design Assets
+
+In this folder you will find the following
+
+- CMSDS Sketch file
+- CMSDS fonts (Bitter, Icons, and Open Sans)
+
+To update the Sketch library version create a pull request with the following: 
+
+1. Update the CMSDS Sketch file
+1. Update the .xml file in the following ways
+    - Update the item `title` with the version number
```

Comment:
When we write our new release scripts, it will be really nice to do this automatically

Reply:
Yes!